### DISCUSSION TITLE
App detection fan curve

### DISCUSSION THREAD

**Discussion by andiramuttakim** [Rem0o/FanControl.Releases#526] [Ideas]
Something like  "target" fan curve that is triggered when specific app is running.

> Idle speed: 20%
> Load speed: 80% 

Useful for long running resource heavy app.

**Comment by Rem0o** [comment]
What's the benefit versus a fan curve that monitor temperatures?

**Reply by andiramuttakim** [reply]
Most of the time, I want my PC to be silent, so I set conservative fan curve.
However sometime, I need to run resource heavy app for long periode of time, at this point I don't care about noise.

So it is basically the fight between aggresive vs conservative fan curve.
This idea will give a head up of what is coming, so the temp will raise slowly, and peak temp won't be that high.

Currently, I'm doing this manually with Flat curve.
